source/tools/hubspot.rst

```diff
@@ -0,0 +1,111 @@
+Hubspot
+============
+
+What is Hubspot?
+----------------
+HubSpot is an "inbound marketing software platform" which offers features for internet marketing, such as emails, blogging and social media. This piece of software is mostly used by the marketing team.
+
+This document will mostly cover tips on how to use the emailer tool/rehabstudio email template.
+
+The rehabstudio email template
+------------------------------
+
+The 2 in 1 template allows you to switch between the event or newsletter email layouts. Each layout includes a spotlight, monthly links and footer section. The spotlight and monthly links can be hidden if not needed.
+
+Name of template:  rehabstudio_new_mailer.html
```

Comment:
@KirMcC filenames should be formatted as inline code (I think using backticks in Markdown, similar to Slack): `rehabstudio_new_mailer.html`